src/xprof_tracer.erl

```diff
@@ -159,7 +159,7 @@ init_tracer() ->
 check_for_overflow(State = #state{status = running,
                                   trace_spec=TraceSpec}) ->
     {_, QLen} = erlang:process_info(self(), message_queue_len),
-    case QLen >= 1000 of
+    case QLen >= application:get_env(xprof, overflow_queue_len, 1000) of
```

Comment:
not really... maybe trace_queue_limit or max_trace_queue_len or trace_queue_overflow_limit ...
(although xprof is all about tracing so the word "trace" is superfluous)